db/migrate/20201006085432_create_redhat_access_telemetry_configuration.rb

```diff
@@ -0,0 +1,20 @@
+class CreateRedhatAccessTelemetryConfiguration < ActiveRecord::Migration[6.0]
```

Comment:
If we decide to use this model (depends on the auth model used), can you rename the file to `20201006085432_create_redhat_access_telemetry_configuration.foreman_rh_cloud.rb` (added `foreman_rh_cloud` label).

Reply:
I think we can exchange this model to a global setting: `Setting[:rh_cloud_disable_insights_proxy]` that will be checked by the `ensure_telemetry_enabled_for_consumer` method.